### DISCUSSION TITLE
Addon Doesn’t work

### DISCUSSION THREAD

**Discussion by Kkrasy** [jacobfortune/Songs-Of-War-Addon#9] [Q&A] [answered]
The addon doesn’t work anymore when will it be fixed

**Comment by lesleyespire** [comment, marked as answer]
Yeah I'm aware, I don't know what problems 1.21 caused for add ons but they're not easy to reverse, you can check this version of the add-on which might still work. https://mcpedl.com/songs-of-war-addon/

**Comment by Prospartan1** [comment]
What is this who is this and what da heck

**Reply by lesleyespire** [reply]
It's a github repo, for a SoW addon, it's quite self-explanatory really...

**Reply by lesleyespire** [reply]
Also you are only receiving these messages since you are subscribed to this thread.

Yours in Sincerity.
